src/commands/dummy.ts

```diff
@@ -0,0 +1,17 @@
+// import * as express from "express";
```

Comment:
Why comment out this file?  I think it makes more sense to leave it executable as the equivalent of a "hello world"